packages/circuit-ui/tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "noUnusedParameters": true,
     "outDir": "./dist/es",
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
What made you add it back?

Reply:
TypeScript was throwing obscure type errors in `node_modules`. I tried to remove it again (now that I'm done with fixes) and it looks fine: reverting